app/core/store/modules/layoutChrome.js

```diff
@@ -0,0 +1,56 @@
+export default {
+  namespaced: true,
+
+  state () {
+    // TODO: Currently saving volume to session instead of database.
+    let cachedSound
+    if (window.sessionStorage) {
+      cachedSound = window.sessionStorage.getItem('layoutChrome/soundOn')
+    }
+
+    return {
+      soundOn: cachedSound !== 'false',
+      currentCourseInstanceId: null,
+      currentCampaignId: null
+    }
+  },
+
+  mutations: {
+    toggleSound (state) {
+      Vue.set(state, 'soundOn', !state.soundOn)
```

Comment:
If you're setting simple keys you don't need to use Vue.set, you can simply do `state.soundOn = !state.soundOn`.  Vue.set is only necessary in a few situations, the most common being creating a new properly on an object.

https://vuex.vuejs.org/guide/mutations.html#mutations-follow-vue-s-reactivity-rules